examples/src/main/java/org/apache/spark/examples/sql/streaming/JavaStructuredKerberiedKafkaWordCount.java

```diff
@@ -0,0 +1,129 @@
+ *      --conf \
+ *      "spark.executor.extraJavaOptions=-Djava.security.auth.login.config=./kafka_jaas.conf" \
+ *      --master yarn
+ *      sql.streaming.JavaStructuredKerberiedKafkaWordCount broker1-host:port,broker2-host:port \
+ *      subscribe topic1,topic2
+ *   Yarn cluster:
+ *    $ bin/run-example --files \
+ *      ${jaas_path}/kafka_jaas.conf,${keytab_path}/kafka.service.keytab,${krb5_path}/krb5.conf \
+ *      --driver-java-options \
+ *      "-Djava.security.auth.login.config=./kafka_jaas.conf \
+ *      -Djava.security.krb5.conf=./krb5.conf" \
+ *      --conf \
+ *      "spark.executor.extraJavaOptions=-Djava.security.auth.login.config=./kafka_jaas.conf" \
+ *      --master yarn --deploy-mode cluster \
+ *      sql.streaming.JavaStructuredKerberiedKafkaWordCount broker1-host:port,broker2-host:port \
+ *      subscribe topic1,topic2
+ *
+ * kafka_jaas.conf can manually create, template as:
+ *   KafkaClient {
+ *     com.sun.security.auth.module.Krb5LoginModule required
+ *     keyTab="./kafka.service.keytab"
+ *     useKeyTab=true
+ *     storeKey=true
+ *     useTicketCache=false
+ *     serviceName="kafka"
+ *     principal="kafka/host@EXAMPLE.COM";
+ *   };
+ * kafka_driver_jaas.conf (used by yarn client) and kafka_jaas.conf are basically the same
+ * except for some differences at 'keyTab'. In kafka_driver_jaas.conf, 'keyTab' should be
+ * "${keytab_path}/kafka.service.keytab".
+ * In addition, for IBM JVMs, please use 'com.ibm.security.auth.module.Krb5LoginModule'
+ * instead of 'com.sun.security.auth.module.Krb5LoginModule'.
+ *
+ * Note that this example uses SASL_PLAINTEXT for simplicity; however,
+ * SASL_PLAINTEXT has no SSL encryption and likely be less secure. Please consider
+ * using SASL_SSL in production.
+ */
+public class JavaStructuredKerberiedKafkaWordCount {
+  public static void main(String[] args) throws Exception {
+    if (args.length < 3) {
+      System.err.println("Usage: JavaStructuredKerberiedKafkaWordCount <bootstrap-servers> " +
+        "<subscribe-type> <topics>");
+      System.exit(1);
+    }
+
+    String bootstrapServers = args[0];
+    String subscribeType = args[1];
+    String topics = args[2];
+
+    SparkSession spark = SparkSession
+      .builder()
+      .appName("JavaStructuredKerberiedKafkaWordCount")
+      .getOrCreate();
+
+    // Create DataSet representing the stream of input lines from kafka
+    Dataset<String> lines = spark
+      .readStream()
+      .format("kafka")
+      .option("kafka.bootstrap.servers", bootstrapServers)
+      .option(subscribeType, topics)
+      .option("kafka.security.protocol", SecurityProtocol.SASL_PLAINTEXT.name)
+      .load()
+      .selectExpr("CAST(value AS STRING)")
+      .as(Encoders.STRING());
+
+    // Generate running word count
+    Dataset<Row> wordCounts = lines.flatMap(
+      (FlatMapFunction<String, String>) x -> Arrays.asList(x.split(" ")).iterator(),
```

Comment:
+1 on fixing this nit.

Reply:
@gaborgsomogyi thanks, others have been fixed. Can you give more information here.

Reply:
I just agree that you've fixed this nit, no action needed.